### PR TITLE
fix: tempature sensor configuration for DS18B20

### PR DESCRIPTION
**Pull request by cellcortex**
fixing up the config for the DS18B20 temperature sensor. 
* The include of the Dallas drivers was before the config import - so it wouldn't pickup the precompiler switch
* There was a typo in the PIN constant name

I also removed a bit of code duplication in the `refreshTemp`

### CODE REVIEW

**Comment by LoQue90** [comment]
anybody who using a DS18B20 sensor is realy welcome to test this PR

File: src/main.cpp [hunk 1]
```diff
@@ -92,7 +94,6 @@ int connectmode = CONNECTMODE;
 
 int offlineMode = 0;
 const int OnlyPID = ONLYPID;
-const int TempSensor = TEMPSENSOR;
```
Comment (LoQue90) [inline src/main.cpp:95]: as far as I know we need the TEMPSENSOR constant macro as a int in the code, so we can change it thru the webseite later

**Comment by murmeltier08** [comment]
Also a test with a normal TSIC will be necessary.

**Comment by LoQue90** [comment]
we gona add a HAL which makes sensor support much easier.
Lets fix ds18B20 sensor after this

**Comment by LoQue90** [comment]
#458 will handleds18B20 support, lets check your fixes after adding the HAL

**Comment by LoQue90** [comment]
should be fix with #458

**Pull request closed without merging**